src/protocol/modulus_conversion/convert_shares.rs

```diff
@@ -156,7 +156,7 @@ pub async fn convert_shares_for_a_bit<F: Field>(
                 packed_bits: *row,
             })
             .execute_one_bit(
-                ctx.narrow(&ModulusConversion(record_id)),
+                ctx,
```

Comment:
@martinthomson made this commit to demo ModulusConversion step is failing when called for different records. Let us know how to best resolve this. The sort test fails with assert "Fails with assert : panicked at 'Refined 'protocol/sort0/mod_conv' with step 'double_random' twice'"

Reply:
Oh.  Looking at your code, it is basically perfect in this regard, so I'll have to look at the modulus conversion stuff now...